src/app/(BottomTabNavigation)/Cases/CaseScreen/EligibilityForm/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import styles from './styles';
+import { updateCaseStatus } from '../../../../../supabase/queries/cases';
+import { CaseUid, Eligibility } from '../../../../../types/types';
+import Arrow from '../../../../../../assets/arrow.svg';
+
+export default function EligibilityForm() {
+  const router = useRouter();
```

Comment:
Same comment as above but let's import `router` instead of `useRouter` so we can remove this line